src/signframework/sign_sha256/sign_sha256.c

```diff
@@ -0,0 +1,630 @@
+/* Copyright 2019 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * 	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* This program sign an input SHA-256 digest with an RSA key
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <time.h>
+
+#include "cca_functions.h"
+#include "cca_structures.h"
+#include "ossl_functions.h"
+#include "utils.h"
+#include "debug.h"
+
+/* local prototypes */
+
+int GetArgs(const char **outputBodyFilename,
+            const char **usr,
+            const char **password,
+            const char **projectLogFileName,
+            const char **sender,
+            const char **project,
+            const char **keyFileName,
+            const char **inputAttachmentFileName,
+            const char **outputAttachmentFileName,
+            unsigned int *bitSize,
+            int *text,
+            int *verbose,
```

Comment:
you removed rsaassa mode. Is that intentional?

Reply:
I started with the sha384 version that doesn't have that.  I could have added that mode but didn't want to spend the time to figure out how to test it